coalib/output/ConsoleInteraction.py

```diff
@@ -199,6 +199,112 @@ def acquire_actions_and_apply(console_printer,
             break
 
 
+def get_context(console_printer, file_dict,
+                sourcerange):  # pragma: no cover
+    """
+    Gets context for the affected lines and also the line numbers
+    of the context
+
+    :param file_dict:       A dictionary containing all files as values with
+                            filenames as key.
+    :param sourcerange:     The SourceRange object referring to the related
+                            lines to print.
+
+    :yield:                 Affected file's contextual information
+
```

Comment:
please format the docstring properly ;) No redundant newlines, and correct alignment. Though rather use this style:
```
:param a:
    xxx
:param b:
    yyy
```
Also I believe `:yield:` is no valid specifier, there's just `:returns:` / `:return:`